core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/EipChecksIT.java

```diff
@@ -32,14 +34,24 @@
 public class EipChecksIT {
 
 	private String versionToTest;
+	private String runtimeProvider;
 
-	public EipChecksIT(final String versionToTest) {
+	public EipChecksIT(final String versionToTest, final String runtimeProvider) {
```

Comment:
I've done machinally as the other parameter was using 'final'.
Effectively clearer without it